# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/jsapi/PeerConnectionMedia.cpp
# Commit: e0a2e54e4de8
# Full Hash: e0a2e54e4de80813829cbcf9e4a5e4ff1285d451
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 9
# Description:
#   Bug 1654112 - Move WebRtcCallWrapper to dedicated files. r=ng
#   
#   This patch also renames it WebrtcCallWrapper, to better match neighboring
#   classes starting on "Webrtc".
#   
# ==============================================================================

diff -r 1965757ee924 -r e0a2e54e4de8 dom/media/webrtc/jsapi/PeerConnectionMedia.cpp
--- a/dom/media/webrtc/jsapi/PeerConnectionMedia.cpp	Thu Sep 02 13:22:03 2021 -0500
+++ b/dom/media/webrtc/jsapi/PeerConnectionMedia.cpp	Thu Apr 08 11:31:43 2021 +0200
@@ -29,6 +29,7 @@
 #include "mozilla/net/NeckoChild.h"
 #include "mozilla/net/WebrtcProxyConfig.h"
 #include "MediaManager.h"
+#include "libwebrtcglue/WebrtcCallWrapper.h"
 #include "libwebrtcglue/WebrtcGmpVideoCodec.h"
 
 namespace mozilla {
@@ -659,10 +660,10 @@
     webrtc::WebRtcKeyValueConfig* aTrials,
     RefPtr<TransceiverImpl>* aTransceiverImpl) {
   if (!mCall) {
-    mCall = WebRtcCallWrapper::Create(
+    mCall = WebrtcCallWrapper::Create(
         mParent->GetTimestampMaker(),
         MakeUnique<media::ShutdownBlockingTicket>(
-            u"WebRtcCallWrapper shutdown blocker"_ns,
+            u"WebrtcCallWrapper shutdown blocker"_ns,
             NS_LITERAL_STRING_FROM_CSTRING(__FILE__), __LINE__),
         aSharedWebrtcState, aTrials);
   }